http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreterSpec.scala

```diff
@@ -31,6 +31,9 @@ import org.scalatest.flatspec.AnyFlatSpecLike
 
 import java.nio.file.Files
 import java.util.Base64
+
+import org.broadinstitute.dsde.workbench.leonardo.AppRestore.GalaxyRestore
```

Comment:
Why was this import needed?

Reply:
Without it, [this](https://github.com/DataBiosphere/leonardo/blob/54a6a9d7b83282d604b36407cba6a12251260281/http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreterSpec.scala#L143) line of code fails